src/api/handlers/messages/postMessages.js

```diff
@@ -37,16 +50,26 @@ export default async (req, res) => {
   };
 
   try {
-    db("messages")
-    .insert(message)
-    .then(() => {
-      res.status(201).send(message);
-    })
-  } catch (err) {
-    res.status(500).send(err);
+    await sendMessage(appointment.patientPhoneNumber, messageBody);
+  } catch (error) {
+    res.status(500).send({
+      error,
+      message: "Could not send text reminder"
+    });
+    return;
   }
 
-  // TODO: Send sms message
-  // Not sure if we want to create a transation and only commit once we get confirmation from twilio
-  // Or write the message to the DB and include a "successful" field, updating after twilio response
+  try {
```

Comment:
I think this should be something like
```
try {
  await db("messages").insert(message);
} catch (error) {
  res.status(500).send({
    error,
    message: "Could not save message to database"
  });
  return;
}

res.status(201).send(message);
```
I think you might be mixing sync/async code here. You can verify the message actually inserted if you want too with
```
let insertedId = null;
try {
insertedId = db("messages").insert(message, ["id"])
...
```

Reply:
Would it be the same thing on line 24 then?

Reply:
Oh, yeah, I missed that before. Good catch!

Reply:
You can also use http://knexjs.org/#Builder-returning as a more concise way of defining a return

Reply:
lol I added that earlier... documentation says mysql is supported, mysql says `.returning() is not supported by mysql and will not have any effect.`

Reply:
Well shut my mouth.